lib/push.js

```diff
@@ -177,6 +181,39 @@ module.exports = function (log, db) {
     },
 
     /**
+     * Notifies every device in the account except the current one that the
+     * password was changed
```

Comment:
(at the very least, we should have a comment here about why the current device doesn't need to be notified)